### PR TITLE
Add report for Sharppy

### PR DESCRIPTION
**Pull request by RJ722**

### CODE REVIEW

**Comment by jendrikseipp** [comment]
The report should state how you came up with the false positives. Also, it would be great to have a third list for items where we're not sure whether they're true or false positives.

**Comment by RJ722** [comment]
Thanks for the hint Jendrik! I've updated the report.

> The report should state how you came up with the false positives. 

The judgement was more or less based on method's name, parent class, where it is used (or other pedestrian ideas including looking for whether or not does it appear in any commented out source code) etc.

File: Sharppy/sharppy_analysis.md [hunk 1]
```diff
@@ -0,0 +1,380 @@
...
+sharppy/viz/vrot.py:274: unused function 'mouseDoubleClickEvent' (60% confidence)
+utils/async.py:81: unused variable 'prio' (60% confidence)
+utils/frozenutils.py:15: unused function 'freezeSupport' (60% confidence)
+```
+
+#### True Positives
+
+After inspecting the source code detected by Vulture as dead, we confirmed that
+the following parts indeed aren't being used anywhere and can therefore be
+deleted safely:
```
Comment (jendrikseipp) [inline Sharppy/sharppy_analysis.md]: Let's phrase this a little differently:

```
#### Probably dead code

After inspecting the source code that Vulture reports as unused, we believe that
the following parts indeed aren't used anywhere and could therefore be deleted:
```

**Review by jendrikseipp — CHANGES_REQUESTED** [review]

File: Sharppy/sharppy_analysis.md (hunk 1 above)
Comment (RJ722) [reply Sharppy/sharppy_analysis.md]: Thanks! The previous one was rather dingy.

**Pull request merged**